circuit_setup/scripts/gen_x509_cert_chain.sh

```diff
@@ -12,17 +12,26 @@ tmpdir=../generated_files/mdl1
 outdir=../inputs/mdl1
 mkdir -p "$tmpdir"
 
+# prevent gitbash path expansion
+export MSYS_NO_PATHCONV=1
+
 # generate self-signed root CA cert
-openssl req -x509 -new -newkey ec:<(openssl ecparam -name secp521r1) -keyout "$tmpdir/root_CA.key" -out "$tmpdir/root_CA.crt" -nodes -subj "/CN=NY DMV Test Root CA" -days 3650 -config openssl_ca.cnf -extensions v3_ca -sha512
+openssl ecparam -name secp521r1 -out tmpkey.pem
```

Comment:
Can you rename tmpkey.pem $tmpdir/ecparamp521?

Reply:
fixed